src/tools/quest_parser/main.cpp

```diff
@@ -258,7 +261,7 @@ void createScript( boost::shared_ptr< Core::Data::Quest >& pQuestData, std::set<
    for( auto enemy : enemy_ids )
    {
       scriptEntry += std::string(
-         "   void onMobKill_" + std::to_string( enemy ) + "( Entity::Player& player )\n"
+         "   void onMobKill_" + std::to_string( enemy ) + "( Entity::Player& player, uint64_t npcId )\n"
```

Comment:
This is incorrect too, the function is `onNpcKill( uint32_t npcId, Entity::Player& player )`

You'll also need to indicate which NPC IDs are to be handled either by outputting a comment in the body of the generated function or by generating a switch/if statement